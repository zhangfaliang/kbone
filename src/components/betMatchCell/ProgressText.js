import React from 'react';
import classnames from 'classnames';
import styles from './index.less';
const ProgressText = ({
  handleClick = () => {},
  stringInTime = '',
  text = '',
  disabled = false,
}) => {
  const handleClickProgress = event => {
    event.preventDefault();
    !disabled && handleClick && handleClick();
  };
  const stringInTimeCls = classnames({
    [styles.stringInTime]: text,
    [styles.stringTime]: !text,
  });

  return (
    <div className={styles.progress} onClick={handleClickProgress}>
      <span className={styles.text}>{text}</span>
      <span className={stringInTimeCls}>{stringInTime}</span>
    </div>
  );
};

ProgressText.COMPONENT_NAME = 'PROGRESSTEXT';
export default ProgressText;
