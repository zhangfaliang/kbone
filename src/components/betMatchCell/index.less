@import '../../theme/index.less';
@defaultHeight:144px;
.push-inplay-bet-cell {
  width: 710px;
  height: @defaultHeight;
  display: flex;
  justify-content: space-between;
  align-items: center;
  background: @bet-group-bg-default-color;
  margin-bottom: 20px;
  border-radius: 8px;
  overflow: hidden;
  box-sizing: border-box;
  .progress {
    font-family: Akrobat-Regular;
    height: @defaultHeight;
    width: 42%;
    display: flex;
    justify-content: flex-end;
    align-items: center;
    padding-right: 20px;
    border-top: 1px solid #D7DAE0;
    flex:1 1 ;
    .text {
      font-size: 28px;
      color: @progress-text-color;
    }
    .stringInTime {
      font-size: 32px;
      color: @progress-string-in-time-color;
      margin-left: 12px;
      font-family: Akrobat-Bold;
    }
    .stringTime{
      font-size: 28px;
      color:#214365;
      margin-left: 12px;
    }
  }
}

.push-before-bet-cell {
  &:extend(.push-inplay-bet-cell all);
  width: 100%;
  margin-bottom: 0;
  height: 146px;
  box-sizing: border-box;
  border-radius: 0px;
}