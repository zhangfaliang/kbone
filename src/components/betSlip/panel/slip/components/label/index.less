.wrap {
  display: flex;
  flex-direction: column;

  .header {
    height: 88px;
  }

  .action {
    display: flex;
    justify-content: space-between;
    padding: 0 24px;
    height: 88px;
    background: #F4F4F5;
    border-radius: 24px 24px 0 0;

    .a {
      display: flex;
      justify-content: space-between;
      align-items: center;
      flex:1;
      .a_a {
        font-family: PingFangSC-Regular;
        font-size: 28px;
        color: #212934;
        letter-spacing: 0;

        span {
          font-size: 56px;
          color: #3D6BFA;
          margin-right: 10px;
        }
      }

      .a_b {
        font-family: PingFangSC-Regular;
        font-size: 28px;
        color: #7A7F85;
        letter-spacing: 0;
        text-align: right;
        display: flex;
      }
    }
  }
}
// @media screen and (min-width: 992px) {
//   .wrap{
//     .header {
//       >div{
//         width: 750px;
//         transform: translateX(0) !important;
//       }
//     }
//   }
 
// }