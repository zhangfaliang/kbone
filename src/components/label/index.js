export { default as LeftTitle } from './LeftTitle.js';
export { default as Label } from './Label.js';
export { default as LabelRight } from './LabelRight.js';
export { default as LabelCenter } from './LabelCenter.js';
export { default as LeftInPlay } from './LeftInPlay';
export { default as HorseLabel } from './horseLabel';
