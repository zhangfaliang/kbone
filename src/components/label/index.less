@import '../../theme/index.less';
@defaultHeigth: 80px;

.default-label {
  height: @defaultHeigth;
  // background: @match-group-bg;
  display: flex;
  width: 100%;
  justify-content: space-between;
  align-items: center;
  font-size: 28px;
  // color: @aide-color-secondary;
  
  // padding-left: @h-p-20;
  padding: 0 24px;
  box-sizing: border-box;
  position: relative;
  // background: #F4F4F5;
 >div{
   &:nth-child(2){
     span {
       color: #BCBEC2;
     }
   }
 }
  &.showTBBorder {
    // border-bottom: 1px solid @tab-bar-border-color;
    // border-top: 1px solid @tab-bar-border-color;
  }
}

.default-left-title {
  display: flex;
  justify-content: flex-start;
  align-items: center;
  flex: 1;

  .gameTypeImg {
    display: flex;
    height: 36px;
    width: 36px;
    resize-mode: contain;
    margin-right: 20px;
  }

  .titleText {
    font-size: 28px;
    // color: @aide-color-secondary;
    width:100%; // 处理订单详情
  }
}

.order-scheme-left-title {
 &:extend(.default-left-title all);
  .titleText {
    font-family: Akrobat-Regular;
    // font-size: 32px;
    // color: @aide-color-secondary;
  }
}

.push-before-left-title {
  &:extend(.default-left-title all);

  .titleText {
    font-size: 28px;
    color: @remark-font-color;
  }

  height: 72px;
  box-sizing:border-box;
  padding-left:@h-p-20;
}

.live-inplay-left-title {
  &:extend(.default-left-title all);
  font-size: 32px;
  font-family: Akrobat-Bold;

  .liveText {
    color: #FFE011;
    margin-right: 10px;

  }

  .inPlayTex {
    color: #fff
  }

}

.default-right-table {
  display: flex;
  justify-content: flex-end;
  align-items: center;
  padding-right: 0;
  box-sizing: border-box;
  // margin-right: 20px;
  .table {
    display: flex;
    justify-content: center;
    align-items: center;
    width: 120px;
    height: 80px;
    font-size: 28px;
    color: #8FB8DF;

  }

  .icon {
    display: flex;
    justify-content: center;
    align-items: center;
    height: @iconHeight;
    width: @iconWidth;

    >span::before {
      // color: @aide-color-secondary;
      color:#BCBEC2; 
    }
  }
}

.index-right-table {
  &:extend(.default-right-table all);

  span:last-child {
    margin-right: 20px;
  }
}

.push-before-right-table {
  &:extend(.default-right-table all);

  .table {
    color: @remark-font-color;
  }

  padding-right:0;

}

.preCompetition-before-right-table {
  &:extend(.default-right-table all);

  .table {
    color: @remark-font-color;
  }

  padding-right:0;

}

.live-inplay-right-table {
  &:extend(.default-right-table all);

  .rightText {
    display: block;
    height: @tab-bar-height;
    text-align: center;
    line-height: @tab-bar-height+1;
    font-size: 28px;
    color: #FFFFFF;
  }

  .icon {
    margin-left: 0px;
  }
}

.push-before-label {
  &:extend(.default-left-title all);
  font-size: 28px;
  color: @remark-font-color;
  height: 72px;
}

.other-match-label {
  &:extend(.default-label all);
  padding-right: 5px;
  font-size: 28px;
  color: @remark-font-color;
  background: #FFFFFF;
  border-radius: 8px;
  margin-top: 20px;
}

.bet-detail-label {
  &:extend(.default-label all);
  font-family: Akrobat-Regular;
  font-size: 32px;
  padding-right: 5px;
  // background: #FFFFFF;
  &.showTBBorder {
    // border-bottom: 1px solid  @bet-border-color;
    // border-top: 1px solid  @bet-border-color;
  }
}

.bet-detail-mt20-label {
  &:extend(.bet-detail-label all);
  margin-top:32px;
  // background: #F5F7FF;
}

.order-detail-label {
  &:extend(.bet-detail-label all);
  height: 72px;
  font-size: 30px;
  padding-right: 20px;
  background: #FFFFFF;
  border: none;
  &.showTBBorder {
    border-bottom: 1px solid  @bet-border-color;
    border-top: 1px solid  @bet-border-color;
  }
}

.default-label-center {
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 28px;
  color: #FFFFFF;
}

.match-detail-label {
  &:extend(.default-label all);
  padding: 0 13px 0 17px;
  color: #FFFFFF;
}

.match-detail-left-title {
  &:extend(.default-left-title all);

  .titleText {
    font-size: 24px;
    color: #FFFFFF;
  }

  flex: none;
}

.other-match-title-left-title {
  &:extend(.default-left-title all);

  .titleText {
    color: #FFFFFF;
  }
}

.white-left-title {
  &:extend(.default-left-title all);

  .titleText {
    color: #FFFFFF;
  }
}

.other-match-item-left-title {
  &:extend(.default-left-title all);

  .titleText {
    color: @remark-font-color;
  }
}

.bet-detail-left-title {
  &:extend(.default-left-title all);
  .titleText {
    font-family: Akrobat-Regular;
    // font-size: 32px;
    font-size: 28px;
    color: @remark-font-color;
    color: #7A7F85;
    margin: 0;
     text-align: left;
  }
}
.bet-detail-mt20-left-title{
  &:extend(.bet-detail-left-title all);
}

.order-detail-left-title {
  &:extend(.default-left-title all);

  .titleText {
    color: @remark-font-color;
  }
}

.match-detail-right-table {
  &:extend(.default-right-table all);

  .icon {
    >span::before {
      color: #fff;
    }
  }

  .activeIcon {
    >span::before {
      color: #FFE011;
    }
  }
}

.bet-detail-right-table {
  &:extend(.default-right-table all);

  .icon {
    margin: 0;

    >span::before {
      font-size: 44px;
    }
  }

}

.order-detail-right-table {
  &:extend(.default-right-table all);

  .rightText {
    color: @remark-font-color;
  }

}
.F5F7FF{
  background: #F5F7FF;
  border-radius: 24px 24px 0 0;
}
.F4F4F5 {
  background: #F4F4F5;
  border-radius: 24px 24px 0 0;
  font-family: SFProText-Regular;
  font-size: 28px;
  color: #7A7F85 ;
  letter-spacing: 0;
}
