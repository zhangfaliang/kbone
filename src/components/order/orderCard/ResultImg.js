import React from 'react';
import styles from './order.less';

const ResultImg = ({ resultImgSrc = '', prefixCls = 'default', alt = '..' }) => {
  return (
    resultImgSrc && (
      <img className={styles[`${prefixCls}-result-img`]} src={resultImgSrc} alt={alt} />
    )
  );
};

ResultImg.COMPONENT_NAME = 'RESULTIMG';

export default ResultImg;
