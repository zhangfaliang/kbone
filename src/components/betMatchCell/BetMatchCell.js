import React, { Component } from 'react';
import styles from './index.less';
import BetMatchInfo from '../betMatchInfo';
import BetButtonCell from '../betButtonCell';
import ProgressText from './ProgressText';
class BetMatchCell extends Component {
  static COMPONENT_NAME = 'BETMATCHCELL';
  handleBetBtn = betInfo => {
    const { matchTime, clickBetBtn } = this.props;
    clickBetBtn({ ...betInfo, matchTime });
  };
  render() {
    const { children, prefixCls } = this.props;
    const classNameStr = `${prefixCls || 'push-inplay'}-bet-cell`;
    const childrens = React.Children.map(children, option => {
      const { clickBetBtn, changeSp, changeHandicap, ...other } = option.props;

      if (option.type.COMPONENT_NAME === 'BETMATCHINFO') {
        return <BetMatchInfo {...other} />;
      } else if (option.type.COMPONENT_NAME === 'BETBUTTONCELL') {
        return (
          <BetButtonCell
            {...other}
            clickBetBtn={this.handleBetBtn}
            changeSp={this.props.changeSp}
            changeHandicap={this.props.changeHandicap}
          />
        );
      } else if (option.type.COMPONENT_NAME === 'PROGRESSTEXT') {
        //handleClick={this.props.betNow}
        return <ProgressText {...other} />;
      }
    });
    return <div className={styles[classNameStr]}>{childrens}</div>;
  }
}
BetMatchCell.defaultProps = {
  matchTime: '',
  betNow: () => {
    console.log('betNow');
  },
  clickBetBtn: params => {
    console.log(params);
  },
  changeSp: () => {
    console.log('changeSp');
  },
  changeHandicap: () => {
    console.log('changeHandicap');
  },
};

export default BetMatchCell;
