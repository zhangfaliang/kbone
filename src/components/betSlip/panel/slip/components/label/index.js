import React, { useState } from 'react';
import classnames from 'classnames';
import { BackHeader } from '@/components/header';
import styles from './index.less';

function Label({
  title,
  isEdit: propsIsEdit,
  desc,
  isLogin,
  onChangeEdit,
  className,
  loginRender,
  noLoginRender,
  onClose,
  betCount,
}) {
  const wrapCls = classnames(className, {
    [styles.wrap]: true,
  });

  const [isEdit, setIsEdit] = useState(propsIsEdit);

  return (
    <div className={wrapCls}>
      <div className={styles.header}>
        <BackHeader
          title="串关投注"
          onClose={onClose}
          rightContent={[isLogin ? loginRender : noLoginRender]}
        />
      </div>
      <div className={styles.action}>
        <div className={styles.a}>
          <div className={styles.a_a}>
            <span>{betCount}</span>项
          </div>
          <div
            className={styles.a_b}
            onClick={() => {
              setIsEdit(!isEdit);
              onChangeEdit(!isEdit);
            }}
          >
            {desc}
          </div>
        </div>
      </div>
    </div>
  );
}

Label.defaultProps = {
  isLogin: true,
  loginRender: 'loginRender',
  noLoginRender: 'noLoginRender',
  onClose: () => {},
};

export default Label;
