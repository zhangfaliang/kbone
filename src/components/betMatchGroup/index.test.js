import toJson from 'enzyme-to-json';
import { render, mount } from 'enzyme';
import { LeftTitle, Label, LabelRight, LabelCenter, LeftInPlay, HorseLabel } from '../label';
import ResultImg from '../order/orderCard/ResultImg';
import { BetMatchCell } from '../betMatchCell';
import BeforeMatchCellGroup from '../beforeMatchCellGroup';
import BetMatchInfo from '../betMatchInfo';
import BetMatchGroup from './index';
import BetButtonCell from '../betButtonCell';
import BetButton from '../betButton';
import Line from '../line';
import Button from '../button';
import { ProgressText } from '../betMatchCell';

describe('betMatchGroup test', async () => {
  it('toJson', () => {
    const wrapper = render(
      <BetMatchGroup>
        <Label>
          <LeftTitle />
          <LabelRight activerightIcon activerightIcon />
          <ResultImg resultImgSrc />
        </Label>

        <BetMatchCell>
          <BetMatchInfo />

          <BetButtonCell>
            <BetButton />
            <Line />
          </BetButtonCell>
        </BetMatchCell>
      </BetMatchGroup>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('HorseLabel', () => {
    const wrapper = render(
      <HorseLabel
        matchTime="2020-03-06 18:30:00'"
        isCountDown
        labels={[{ labelText: '23', isCountDown: true, isBtn: true }]}
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('toJson BeforeMatchCellGroup', () => {
    const wrapper = render(
      <BetMatchGroup>
        <Label showTBBorder={true}>
          <LeftTitle />
        </Label>

        <BeforeMatchCellGroup>
          <Label prefixCls="push-before" showTBBorder={false}>
            <LeftTitle gameTypeImgShow={false} prefixCls="push-before" />
            <LabelRight prefixCls="push-before" />
          </Label>

          <BetMatchCell>
            <BetMatchInfo />

            <BetButtonCell>
              <BetButton />
            </BetButtonCell>
          </BetMatchCell>
        </BeforeMatchCellGroup>
      </BetMatchGroup>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it(' BeforeMatchCellGroup', () => {
    const wrapper = mount(
      <BetMatchGroup>
        <Label showTBBorder={true}>
          <LeftTitle />
          <LeftTitle />
          <LabelCenter />
          <LabelRight arrayText={['1', '3']} showIcon rightText />
          <LeftInPlay />
        </Label>

        <BeforeMatchCellGroup>
          <Label prefixCls="push-before" showTBBorder={false}>
            <LeftTitle gameTypeImgShow={false} prefixCls="push-before" />
            <LabelRight prefixCls="push-before" />
          </Label>

          <BetMatchCell>
            <BetMatchInfo />

            <BetButtonCell>
              <BetButton />
            </BetButtonCell>
          </BetMatchCell>
        </BeforeMatchCellGroup>
        <Button />
      </BetMatchGroup>
    );
    wrapper
      .find('BeforeMatchCellGroup')
      .find('BetMatchCell')
      .find('BetButtonCell')
      .find('BetButton')
      .simulate('click');
    wrapper.find('Button').simulate('click');
  });

  it(' BeforeMatchCellGroup LeftTitle', () => {
    const wrapper = mount(
      <BetMatchGroup>
        <Label showTBBorder={true}>
          <LeftTitle />
          <LabelCenter />
          <LabelRight arrayText={['1', '3']} showIcon rightText />
          <LeftInPlay />
        </Label>

        <BeforeMatchCellGroup>
          <Label prefixCls="push-before" showTBBorder={false}>
            <LeftTitle gameTypeImgShow={false} prefixCls="push-before" />
            <LabelRight prefixCls="push-before" />
          </Label>

          <BetMatchCell>
            <BetMatchInfo />
            <BetButtonCell>
              <BetButton />
            </BetButtonCell>
            <ProgressText text={'ewrwe'} stringInTime={'werwer'} />
          </BetMatchCell>
        </BeforeMatchCellGroup>
        <Button />
      </BetMatchGroup>
    );
    wrapper
      .find('BeforeMatchCellGroup')
      .find('BetMatchCell')
      .find('ProgressText')
      .simulate('click');
    wrapper
      .find('BeforeMatchCellGroup')
      .find('Label')
      .find('LeftTitle')
      .simulate('click');
    wrapper.find('Button').simulate('click');

    const clickLeftTitleText = wrapper
      .find('BeforeMatchCellGroup')
      .find('Label')
      .find('LeftTitle')
      .prop('clickLeftTitleText');
    const click = jest.fn(clickLeftTitleText);
    expect(click()).toBe('clickLeftTitleText');
  });

  it(' BeforeMatchCellGroup LabelRight', () => {
    const wrapper = mount(
      <BetMatchGroup>
        <Label showTBBorder={true}>
          <LeftTitle />
          <LabelCenter />
          <LabelRight arrayText={['1', '3']} showIcon rightText />
          <LeftInPlay />
        </Label>

        <BeforeMatchCellGroup>
          <Label prefixCls="push-before" showTBBorder={false}>
            <LeftTitle gameTypeImgShow={false} prefixCls="push-before" />
            <LabelRight showIcon disabled={false} prefixCls="push-before" />
            <LeftInPlay liveText />
          </Label>

          <BetMatchCell>
            <BetMatchInfo />
            <BetButtonCell>
              <BetButton disabled isDetail={false} optionName="fs" handicap />
            </BetButtonCell>
          </BetMatchCell>
        </BeforeMatchCellGroup>
        <Button />
      </BetMatchGroup>
    );
    wrapper
      .find('BeforeMatchCellGroup')
      .find('Label')
      .find('LeftInPlay')
      .simulate('click');

    wrapper.find('Button').simulate('click');
    wrapper
      .find('BeforeMatchCellGroup')
      .find('Label')
      .find('LabelRight')
      .find('div')
      .at(1)
      .simulate('click');
    wrapper.find('Button').simulate('click');
    const handleRightIcon = wrapper
      .find('BeforeMatchCellGroup')
      .find('Label')
      .find('LabelRight')
      .prop('handleRightIcon');
    const click = jest.fn(handleRightIcon);
    expect(click()).toBe('handleRightIcon');
  });

  it(' BeforeMatchCellGroup', () => {
    const wrapper = mount(
      <BetMatchGroup>
        <Label showTBBorder={true}>
          <LeftTitle />
          <LeftTitle />
          <LabelCenter />
          <LabelRight arrayText={['1', '3']} showIcon rightText />
          <LeftInPlay />
        </Label>

        <BeforeMatchCellGroup>
          <Label prefixCls="push-before" showTBBorder={false}>
            <LeftTitle gameTypeImgShow={false} prefixCls="push-before" />
            <LabelRight prefixCls="push-before" />
          </Label>

          <BetMatchCell>
            <BetMatchInfo />
            <BetButtonCell>
              <BetButton disabled isDetail optionName="fs" handicap />
            </BetButtonCell>
          </BetMatchCell>
        </BeforeMatchCellGroup>
        <Button />
      </BetMatchGroup>
    );
    wrapper
      .find('BeforeMatchCellGroup')
      .find('BetMatchCell')
      .find('BetButtonCell')
      .find('BetButton')
      .simulate('click');
    wrapper.find('Button').simulate('click');
  });
});
