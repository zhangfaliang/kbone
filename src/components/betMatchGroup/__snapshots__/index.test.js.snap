// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`betMatchGroup test HorseLabel 1`] = `
<div
  class="horse-label"
>
  <span
    class="text button"
  >
    23
  </span>
</div>
`;

exports[`betMatchGroup test toJson 1`] = `
<div
  class="default-bet-match-group"
>
  <div
    class="default-label"
  >
    <div
      class="default-left-title"
    >
      <img
        alt=""
        class="gameTypeImg"
        src="https://avatars3.githubusercontent.com/u/17559536?s=40&v=4"
      />
      <span
        class="titleText"
      />
    </div>
    <div
      class="default-right-table"
    />
    <img
      alt=".."
      class="default-result-img"
    />
  </div>
  <div
    class="push-inplay-bet-cell"
  >
    <div
      class="wrap"
    >
      <div
        class="nameOrScore"
      >
        <span
          class="name"
        >
          Villacher Sv
        </span>
      </div>
      <div
        class="nameOrScore"
      >
        <span
          class="name"
        >
          Greifenburg
        </span>
      </div>
      <div
        class="bottom"
      >
        <span
          class="live"
        >
          LIVE
        </span>
        <span
          class="time"
        >
          4th set 10'
        </span>
      </div>
    </div>
    <div
      class="push-inplay-betButtonCell"
    >
      <div
        class="default-bet-button"
      >
        <span
          class="up"
        >
          <span
            class="iconfont icon-sheng"
          />
        </span>
        <span
          class="down"
        >
          <span
            class="iconfont icon-jiang"
          />
        </span>
        <div
          class="wrapSp"
        >
          <span
            class="sp"
          />
        </div>
      </div>
      <div
        class="row-line"
      />
    </div>
  </div>
</div>
`;

exports[`betMatchGroup test toJson BeforeMatchCellGroup 1`] = `
<div
  class="default-bet-match-group"
>
  <div
    class="default-label showTBBorder"
  >
    <div
      class="default-left-title"
    >
      <img
        alt=""
        class="gameTypeImg"
        src="https://avatars3.githubusercontent.com/u/17559536?s=40&v=4"
      />
      <span
        class="titleText"
      />
    </div>
  </div>
  <div
    class="before-match-cell-group"
  >
    <div
      class="push-before-label"
    >
      <div
        class="push-before-left-title"
      >
        <span
          class="titleText"
        />
      </div>
      <div
        class="push-before-right-table"
      />
    </div>
    <div
      class="push-inplay-bet-cell"
    >
      <div
        class="wrap"
      >
        <div
          class="nameOrScore"
        >
          <span
            class="name"
          >
            Villacher Sv
          </span>
        </div>
        <div
          class="nameOrScore"
        >
          <span
            class="name"
          >
            Greifenburg
          </span>
        </div>
        <div
          class="bottom"
        >
          <span
            class="live"
          >
            LIVE
          </span>
          <span
            class="time"
          >
            4th set 10'
          </span>
        </div>
      </div>
      <div
        class="push-inplay-betButtonCell"
      >
        <div
          class="default-bet-button"
        >
          <span
            class="up"
          >
            <span
              class="iconfont icon-sheng"
            />
          </span>
          <span
            class="down"
          >
            <span
              class="iconfont icon-jiang"
            />
          </span>
          <div
            class="wrapSp"
          >
            <span
              class="sp"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
